Sources/GDBRemote/Session.cpp

```diff
@@ -2060,16 +2060,15 @@ void Session::Handle_qProcessInfoPID(ProtocolInterpreter::Handler const &,
 //
 void Session::Handle_qRcmd(ProtocolInterpreter::Handler const &,
                            std::string const &args) {
-  ErrorCode error = _delegate->onExecuteCommand(*this, args);
+  std::string cmd = HexToString(args);
+
+  ErrorCode error = _delegate->onExecuteCommand(*this, cmd);
   if (error != kSuccess) {
     sendError(error);
     return;
   }
 
-  //
-  // No message is sent as a reply since it is sent by the
-  // handled message.
-  //
+  sendOK();
```

Comment:
You can just `sendError(_delegate->onExecuteCommand(*this, cmd));` instead of doing any error checking here.